lib/cli/nodewizardcommand.cpp

```diff
@@ -604,6 +643,45 @@ int NodeWizardCommand::MasterSetup(void) const
 	globalZones.push_back("global-templates");
 	globalZones.push_back("director-global");
 
+	std::cout << "\nDo you want to specify additional global zones? [y/N]: ";
+
+	std::getline(std::cin, answer);
+	boost::algorithm::to_lower(answer);
+	choice = answer;
+
+wizard_global_zone_loop_start:
+	if (choice.Contains("y")) {
+		std::cout << "\nPlease specify the name of the global Zone: ";
+
+		std::getline(std::cin, answer);
+
+		if (answer.empty()) {
+			Log(LogWarning, "cli", "Name of the global Zone is required! Please retry.");
```

Comment:
User-visible error messages should not use Log(). Starting with 2.8 the default log level for `node wizard` is `LogCritical`, i.e. this message would be hidden unless the user specifies `--verbose`.